### ISSUE TITLE
`brew install zig --HEAD` fails to install

### ISSUE THREAD

**Issue by stefangiba** [ziglang/zig#15956]
Hi,

For whatever reason, trying to install a nightly build of Zig via Homebrew on an M1 Mac is failing, as it seems like brew fails to install necessary dependencies. Fixing the issue is pretty easy, as installing the dependencies manually seems to allow the source build to continue, but having the install process work right away would be ideal.

Thanks!

```
$ brew install zig --HEAD                                                                                                                                                                
==> Downloading https://formulae.brew.sh/api/formula.jws.json
####################################################################################################################################################################################### 100.0%
==> Downloading https://formulae.brew.sh/api/cask.jws.json
####################################################################################################################################################################################### 100.0%
==> Fetching zig
==> Downloading https://raw.githubusercontent.com/Homebrew/homebrew-core/2a1d0d3dd7b384c6d7cb16a507009e3d1eb73677/Formula/zig.rb
Already downloaded: /Users/user/Library/Caches/Homebrew/downloads/9d070b58362371523448c15797d8c4818495026222277186c2fc147a49f17171--zig.rb
==> Cloning https://github.com/ziglang/zig.git
Updating /Users/user/Library/Caches/Homebrew/zig--git
==> Checking out branch master
Already on 'master'
Your branch is up to date with 'origin/master'.
HEAD is now at 7e0a02ee2 macos+libc: add zlib.h,sys/paths.h,netinet/{ip.h,udp.h}
==> Installing dependencies for zig: python@3.11, six, z3 and llvm
==> Installing zig dependency: python@3.11
==> Pouring python@3.11--3.11.3.arm64_ventura.bottle.tar.gz
Error: No such file or directory @ rb_sysopen - /Users/user/Library/Caches/Homebrew/downloads/13442ac96116dd964aeed18feaeb64d15f66235eb6c34abe73d1efebf6000713--python@3.11--3.11.3.arm64_ventura.bottle.tar.gz
```

**Comment by devnexen** [comment]
Do you think opening an issue on homebrew instead would be appropriate ?

**Comment by catsby** [comment]
Hello - 

I encountered this error as well when `brew` was trying to install the `six` dependency. I tried installing `six` directly with `brew install six` and it installed the same version with no issue. Re-running `brew install zig --head` lead to the same error but this time on installing `z3`. I then installed `z3` with `brew install` with no issue. _After that_, `brew install zig --head` worked fine. 

I realize this does not address the origin of the issue but thought I'd share for anyone else that lands here coming from an internet search.
